README.md

```diff
@@ -17,7 +17,10 @@ Ce programme permet de valider et de normaliser les données présentes dans une
 
 Le paramétrage s'effectue à l'aide de [fichiers JSON décrivant des arborescences de fichiers et des tables](https://github.com/IGNF/validator-schema#ignfvalidator-schema).
```

Comment:
Link to be changed

Reply:
Fixed to validator-core/src/main/resources/schema/README.md , thanks!